### ISSUE TITLE
Unable to sacrifice units with the Soothsayer

### ISSUE THREAD

**Issue by the-m4a** [civfanatics/CQUI_Community-Edition#28]
Investigate:
From the Workshop page:

This mod makes it so I'm unable to sacrifice units with the soothsayer during the Appease The Gods emergency. This is the only bug I've come across so far, but it is annoying having to disable the mod whenever the emergency pops up

**Comment by the-m4a** [comment]
Reproduced this issue - click on the button for "Sacrifice Adjacent Unit" and nothing happens.  Looks like there's some stuff in WorldInput.lua and UnitPanel.lua to integrate

**Comment by the-m4a** [comment]
oof.  worldinput.lua needs some work.  lots of differences in the CQUI version and the in-game version

**Comment by the-m4a** [comment]
In spite of that needed work, it was at least fairly easy to enable the Sacrifice stuff.  It seems to work, though I did not get a significant chance to test it.
I started up a game as the Maya in Apocolypse mode.  Let it autoplay for 50 turns, then about 20 turns at a time until I finally got to a point where an Appease the Gods emergency was going on.  Dropped a soothsayer and couple of units next to Kilamenjaro and was able to sacrifice.
The score of the emergency doesn't update immediately, but it does the next turn (which is fairly standard for those things I think).

The code changes are in this branch - no PR yet, as I want to ensure the other PR gets done first.
https://github.com/civfanatics/CQUI_Community-Edition/tree/SoothsayerSacrificeFix
